src/lib/Libifl/pbsD_submit.c

```diff
@@ -164,6 +166,8 @@ __pbs_submit(int c, struct attropl  *attrib, char *script, char *destination, ch
 	struct cred_info *cred_info = NULL;
 	int commit_done = 0;
 	char *lextend = NULL;
+	svr_conn_t *svr_connections = get_conn_servers(c);
```

Comment:
would it be lesser confusion if we rename the function get_conn_servers() to get_conn_svr_instances()? That might re-enforce the fact that these are merely server instances?